mock-echo-app/src/cmr/mock_echo/client/echo_util.clj

```diff
@@ -79,7 +114,7 @@
   ([provider-guid coll-identifier]
    (catalog-item-id provider-guid coll-identifier nil))
   ([provider-guid coll-identifier gran-identifier]
-   {:provider-guid provider-guid
+   {:provider_id provider-guid
```

Comment:
I thought access control would accept both _ and -, but changing to dashes caused:
{:status 400,
 :body
 {:errors
  ["object instance has properties which are not allowed by the schema: [\"group-permissions\",\"system-identity\"]"
   "instance failed to match exactly one schema (matched 0 out of 4)"
   "object has missing required properties ([\"group_permissions\",\"system_identity\"])"
   "object has missing required properties ([\"group_permissions\",\"provider_identity\"])"
   "object has missing required properties ([\"group_permissions\",\"single_instance_identity\"])"
   "object has missing required properties ([\"catalog_item_identity\",\"group_permissions\"])"]},
 :content-type :json}